frontend/plugins/filters.js

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
+
+var numeral = require("numeral");
+
+Vue.filter("formatNumber", function (value) {
+  return numeral(value).format("0,0"); // displaying other groupings/separators is possible, look at the docs
```

Comment:
We should accept language dependant formatting. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

Reply:
I have included format based on locale